plugins/priceFeed.go

```diff
@@ -42,6 +44,13 @@ func MakePriceFeed(feedType string, url string) (api.PriceFeed, error) {
 		return newFiatFeed(url), nil
 	case "fiat-oxr":
 		return newFiatFeedOxr(url), nil
+	case "generic-price-feed":
+		parts := strings.Split(url, ";")
```

Comment:
Can we move the extraction of the url parts into the factory method. So it should just be one call:

makeGenericPriceFeed(url)

like we do for the other feeds. What do you think?

Reply:
Logic for delimiter seems reasonable